README.md

```diff
@@ -69,8 +69,8 @@ Mastodon acts as an OAuth2 provider, so 3rd party apps can use the REST and Stre
 
 - **PostgreSQL** 12+
 - **Redis** 4+
-- **Ruby** 3.1+
-- **Node.js** 16+
+- **Ruby** 3.2+
+- **Node.js** 18+
```

Comment:
In line with [`package.json`](https://github.com/mastodon/mastodon/blob/85fdbd0ad53837c9209acf3fb45811d5bae41cd9/package.json#L6).